howso/feature_residuals.amlg

```diff
@@ -620,7 +620,7 @@
 										;ensure to zip a list of classes by prepending an empty list
 										(append (list) (current_value))
 										;zip all classes with a value of 1 so that all clases in the predicted list are assigned a value of 1
-										1
+										(range (lambda (replace 1)) 1 (size (current_value)) 1)
```

Comment:
The copy even makes a difference for this? A primitive declared in a lambda? The amount of these types of changes we're making would suggest there's language improvements to be made since this doesn't feel like a sustainable pattern to encourage...

Reply:
At the very least, it's not a particularly intuitive/readable way to code.